sdk/healthinsights/azure-health-insights-cancerprofiling/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+- Initial preview of the Azure Health Insights CancerProfiling client library.
+
+### Features Added
```

Comment:
You will need to delete those tabs which have empty content.

Reply:
done